### ISSUE TITLE
newline-after-import not recognizing decorator

### ISSUE THREAD

**Issue by hannahjin** [benmosher/eslint-plugin-import#1004]
I set my eslint config to have 2 new lines after imports using `newline-after-import` plugin. We are also allowing decorator syntax with `@`.
Currently, it is not counting decorator as a line of code. Only one new line appears after import group if the code block starts with a decorator. 

Currently
```
import pick from 'ramda'

@withRouter
export default class MyComponent extends React.Component {}
```

Want 
```
import pick from 'ramda'


@withRouter
export default class MyComponent extends React.Component {}
```

**Comment by ljharb** [comment]
Presumably:
```jsx
import pick from 'ramda'
export default class MyComponent extends React.Component {}
```
autofixes to:
```jsx
import pick from 'ramda'


export default class MyComponent extends React.Component {}
```
?

If so, this seems like a bug.

**Comment by hannahjin** [comment]
In the above case, it works as expected and autofixes to 2 new lines after import. However, when there's a decorator before the export, there is only one blank line between the import and the decorator.

**Comment by ljharb** [comment]
@atos1990 that seems like a good test case and fix - want to submit a PR?

**Comment by atos1990** [comment]
@ljharb, yes, no problem

**Comment by waitingsong** [comment]
I have the same problem with options
```
  import/newline-after-import: [2, { "count": 2 } ]
```
code:
```ts
import { Context, config, controller, get, provide } from 'mymodule'
@provide()
@controller('/')
export class HomeController {
.....
.....
}

```

**Comment by TrejGun** [comment]
hey guys, this is still an issue! can we have this fixed?
